File: feature-promo/src/main/java/com/levnovikov/feature_promo/promo_list/PromoListPresenter.java
package com.levnovikov.feature_promo.promo_list;

import com.levnovikov.feature_promo.domain.Promo;

import java.util.List;

/**
 * Author: lev.novikov
 * Date: 4/1/18.
 */

public interface PromoListPresenter {

    void setData(List<Promo> promoList);
}

File: feature-promo/src/main/java/com/levnovikov/feature_promo/promo_list/PromoListPresenter.kt
package com.levnovikov.feature_promo.promo_list

import com.levnovikov.feature_promo.domain.Promo

/**
 * Author: lev.novikov
 * Date: 4/1/18.
 */

interface PromoListPresenter {

    fun setData(promoList: List<Promo>)
}
